curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

```diff
@@ -17,439 +17,614 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+In which scenario is using an attribute selector a bad idea?
 
 #### --distractors--
 
-Placeholder distractor 1
+```css
+input[type="text"]:focus {
+    background-color: yellow;
+}
+```
 
 ---
 
-Placeholder distractor 2
+```css
+img[alt] {
+    border: 2px solid red;
+}
+```
 
 ---
 
-Placeholder distractor 3
+```css
+input[type="text"] {
+    border: 1px solid gray;
+}
+```
 
 #### --answer--
 
-Placeholder answer
+```css
+a[href^="http"] { 
```

Comment:
Why is this bad?

Feels like a subjective thing.

Reply:
The idea is that it will highlight all links on the page. Compared to other options this seems unnecessary. Also, if a developer does want a formatting in all the links, they can do so in simpler ways. 

Although on a second thought, this can have a use-case since all links may not be http.

I'll change this question.

Reply:
Hello, I have replaced this question with a new one.